pcdsdevices/elliptec.py

```diff
@@ -0,0 +1,62 @@
+"""
+Classes for ThorLabs Elliptec motors.
+"""
+
+
+from ophyd import Component as Cpt
+from ophyd import Device
+from ophyd.signal import EpicsSignal, EpicsSignalRO
+
+
+class ELL_Base(Device):
+    """
+    Base class for Elliptec stages.
+
+    Sufficient for control of ELL17/20 (28/60mm linear stages) and ELL14/18
+    (rotation) stages.
+
+    Parameters
+    ----------
+    prefix : str
+        The PV base of the stage.
+    """
+    _Target_Position = Cpt(EpicsSignal, ':MOVE', name='Target Position',
+                           kind='normal')
+    _Target_Precision = Cpt(EpicsSignal, ':MOVE.PREC', name='Target Precision',
+                            kind='config')
+    _Target_EGU = Cpt(EpicsSignal, ':MOVE.EGU', name='Target EGU',
+                      kind='config')
+    _Current_Position = Cpt(EpicsSignalRO, ':CURPOS', name='Current Position',
+                            kind='normal')
+    _Current_Precision = Cpt(EpicsSignal, ':CURPOS.PREC',
+                             name='Current Precision', kind='config')
+    _Current_EGU = Cpt(EpicsSignal, ':CURPOS.EGU', name='Current EGU',
```

Comment:
Renaming signals here can have undesirable consequences in `bluesky` scans, where any two devices of this type will have exactly the same data keys for the corresponding values.